src/gpgpu-sim/shader.cc

```diff
@@ -448,9 +448,9 @@ void shader_core_ctx::create_exec_pipeline() {
     }
   }
 
-  m_ldst_unit = new ldst_unit(m_icnt, m_mem_fetch_allocator, this,
+  m_ldst_unit = new ldst_unit(m_gpu, m_icnt, m_mem_fetch_allocator, this,
```

Comment:
the ldst unit now needs the whole GPU because it needs to look up some global page table information?